test/CMakeLists.txt

```diff
@@ -321,20 +321,6 @@ if(MIOPEN_TEST_MLIR)
     set(IMPLICITGEMM_MLIR_ARGS_F ${IMPLICITGEMM_ARGS} --verbose --disable-backward-data --disable-backward-weights)
     set(IMPLICITGEMM_MLIR_ARGS_B ${IMPLICITGEMM_ARGS} --verbose --disable-forward --disable-backward-weights)
     set(IMPLICITGEMM_MLIR_ARGS_W ${IMPLICITGEMM_ARGS} --verbose --disable-forward --disable-backward-data)
-    
-    add_custom_test(test_conv_igemm_mlir ALLOW_NONXDLOPS
```

Comment:
Are these tests necessary for MLIR CPP backend only?

Reply:
The way I have it as now will make it run into merge conflict with #841. Once #841 is merged, I will rework this to get rid of cpp solver only, and keep bin solvers.

Reply:
Okay